docs/bugs.json

```
@@ -1,4 +1,14 @@
 [
+    {
+        "uid": "SOL-2025-4",
```

It's not a month. It's a counter :)

```suggestion
        "uid": "SOL-2025-1",
```

---

what does it count? The number of bugs in the year?

---

Yes. It's just a separate counter that resets when the year changes.